src/main/java/fr/esgi/al5_2/Tayarim/dto/proprietaire/AdministrateurCreationDTO.java

```diff
@@ -0,0 +1,38 @@
+package fr.esgi.al5_2.Tayarim.dto.proprietaire;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data // Generates getters, setters, toString, equals, and hashCode methods
+@RequiredArgsConstructor
+@AllArgsConstructor // Generates an all-args constructor
+@Builder
+public class AdministrateurCreationDTO {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'AdministrateurCreationDTO' must contain no more than '1' consecutive capital letters.